src/Generator.Rewrite/Program.cs

```diff
@@ -804,6 +842,72 @@ private static GeneratedVariableIdentifier EmitStringBuilderParameter(MethodDefi
             return generatedVariables;
         }
 
+        private class CountAttribute
```

Comment:
Not super keen on having multiple classes in one file, especially since this is something we'll be going back and fixing in the rest of the codebase. Possible to make internal?